python/ray/dag/compiled_dag_node.py

```diff
@@ -45,7 +45,7 @@ def do_allocate_channel(
 
     Args:
         readers: The actor handles of the readers.
-        buffer_size_bytes: The maximum size of messages in the channel.
```

Comment:
typo? type?

Reply:
No. `typ` follows the naming convention of Ray codebase.

Reply:
ok, seems we only call it `typ` in ADAG codebase tho.